test/test_filo_async.c

```diff
@@ -65,27 +67,50 @@ int main(int argc, char* argv[])
     int rank_from_file_name = *((strstr(dst_filelist[i], ".out"))-1) - '0';
     //assertain that the filename with consistant process marker was passed through flush/fetch
     if(rank != rank_from_file_name){
-      rc = TEST_FAIL;
+      printf ("Error in line %d, file %s, function %s.\n", __LINE__, __FILE__, __func__);
       printf("rank = %d, rank_from_file_name = %d\n", rank, rank_from_file_name);
+      return TEST_FAIL;
     }
     //assertain that the file content is consistent with the process
     FILE *file = fopen(dst_filelist[i], "r");
     char *readContent = NULL;
     size_t readContent_size = 0;
     if (!file){
-      rc = TEST_FAIL;
+      printf ("Error in line %d, file %s, function %s.\n", __LINE__, __FILE__, __func__);
       printf("Error opening file %s: %d %s\n", dst_filelist[i] , errno, strerror(errno));
+      return TEST_FAIL;
     }
     size_t line_size = getline(&readContent, &readContent_size, file);
     if (strcmp(readContent, proc_specific_file_content) != 0){
-      rc = TEST_FAIL;
+      printf ("Error in line %d, file %s, function %s.\n", __LINE__, __FILE__, __func__);
       printf("flushed file content = %s, fetched file content = %s\n", proc_specific_file_content, readContent);
+      return TEST_FAIL;
     }
     fclose(file);
 
     free(src_filelist[i]);
     free(dst_filelist[i]);
   }
+  //MPI_COMM_NULL test
+  rc = Filo_Flush_start("mapfile", NULL, 1, filelist, dest_filelist, MPI_COMM_NULL, "pthread");
+  if(rc == FILO_SUCCESS){
+    printf ("Error in line %d, file %s, function %s.\n", __LINE__, __FILE__, __func__);
+    printf("Filo_Flush_start succeded with MPI_COMM_NULL comm parameters\n");
```

Comment:
Same type three times here:
```suggestion
    printf("Filo_Flush_start succeeded with MPI_COMM_NULL comm parameters\n");
```